src/api/sars-cov-2-resolvers.ts

```diff
@@ -0,0 +1,77 @@
+import { MongoClient } from "mongodb";
+import { SarsCov2Estimate, QueryResolvers } from "./graphql-types/__generated__/graphql-types";
+import { ArbovirusEstimateDocument, SarsCov2EstimateDocument } from "../storage/types";
+
+interface GenerateSarsCov2ResolversInput {
+  mongoClient: MongoClient;
+}
+
+interface GenerateSarsCov2ResolversOutput {
+  sarsCov2Resolvers: { Query: QueryResolvers }
+}
+
+const filterUndefinedValuesFromArray = <T>(array: (T | undefined)[]): T[] => array.filter((element): element is T => !!element);
+
+const transformSarsCov2EstimateDocumentForApi = (document: SarsCov2EstimateDocument): SarsCov2Estimate => {
+  return {
+    id: document._id.toHexString(),
```

Comment:
why do we do this? Is it to reduce an extra layer in the object so instead of like apiResponse[index].document.ageGroup, we would just do apiResponse[index].ageGroup?

Reply:
Not quite, it's because sometimes the name in the database doesn't match the name on the API (Ex. `_id` vs `id`, `_id`) and sometimes the data type in the database doesn't match the type needed by the API (Ex. `_id` is an `ObjectId` object and I need to convert it to a string, `samplingEndDate` is a JS `Date` object but I need an ISO timestamp for the API.